src/index.js

```diff
@@ -234,6 +234,11 @@ const createListItem = (location, status, lng, lat) => {
     seekingVolunteers = `<span data-translation-id="seeking_volunteers" class="seekingVolunteersBadge location-list--badge">Needs Volunteer Support</span>`
   }
 
+  let covid19Testing = ''
+  if (location.notes && location.notes.match(/(?:\bcovid-19 testing\b)/i)) {
```

Comment:
```suggestion
  if (location.notes && location.notes.match(/(?:\bcovid(-19)? test(ing)?\b)/i)) {
```

I like where you're going with this! I was thinking there might be some other similar ways a user could define this that we might want to consider. This updated regex makes `-19` optional as well as `ing` at the end of testing optional so that the regex now matches:

* covid test
* covid testing
* covid-19 test
* covid-19 testing

Reply:
Hi @crhistianramirez,  this is a good point -  the reason I constrained it to just the string "covid-19 testing" was to eliminate false positives, and because I have some control over the data entry side. I'm worried that if a site put "Please get a covid-19 test before volunteering here" our regex would pick them up as a testing site.